k8s/tf/70_prod_db_redis/main.tf

```diff
@@ -19,13 +19,13 @@ module "mysql-prod" {
   mysql_username              = "root"
   mysql_password              = var.mysql_prod_password
   mysql_identifier            = "sumo"
-  mysql_instance_class        = "db.m5.4xlarge"
+  mysql_instance_class        = "db.m5.2xlarge"
   mysql_backup_retention_days = 14
   mysql_security_group_name   = "sumo_rds_sg_prod"
   mysql_storage_gb            = 250
   mysql_storage_type          = "gp2"
   mysql_engine_version = {
-    mysql = "5.6.51"
+    mysql = "5.7"
```

Comment:
Shouldn't we specify the full version here? The latest minor release on RDS is 5.7.34.

Reply:
With `auto_minor_version_upgrade` set as **true** `5.7` must be enough.

Reply:
I was under the impression that we will get diffs, but seems reasonable anyway. We would get a diff as soon as the next minor version upgrade happens, and it's not useful to keep up with that.